packages/odyssey-react/src/components/Field/Field.tsx

```diff
@@ -139,15 +149,15 @@ function Label(props: PropsLabel) {
           </Text>
         </span>
       )}
-    </Tag>
+    </Box>
   );
 
   const labelVisuallyHidden = (
     <ScreenReaderText>
-      <Tag htmlFor={inputId}>
+      <Box as={as} htmlFor={inputId}>
         {children}
         {!required && optionalLabel && <span children={optionalLabel} />}
-      </Tag>
+      </Box>
```

Comment:
Can you talk me thru why `Hint` and `FieldError` below are not being updated at all?

I see they are both using a wrapping `p` tag that applies margin from classes written by hand `styles.hint` and `styles.error`. They could potentially be replaced by Box and use margin props, however I'm not sure how the existing `<Text />` components do or do not account for styling children correctly already...

Reply:
Maybe:

```
<Box as="p" className={styles.hint} id={`${id}-hint`}>
  {children}
</Box>
```

TBH I'm still unclear on whether we want to use `Box` for only the Component root or also for Component.Part roots.

Reply:
@edburyenegren-okta using Box (or Text) as the root of a `Component.Part` would be useful if it was ever rendered in isolation. This is very much possible but not immediately obvious in our design of adding Part as a static component on Component. Typically Part is rendered as a child of Component so if Box (or Text) already exists at the root of Component any values also rendered by Part will pick up the inherited styles unless they are overriden. Like I mentioned above if Part is rendered elsewhere outside this context it may be good to be defensive and have Box (or Text) as the root to account for that.

Reply:
Related to my original question above and re-using Box margin styles, this looks like another example of the "utility class" [discussion here.](https://github.com/okta/odyssey/pull/1214/files/a6f094c668acf1749a42619ffc2cd016e9999410#r770955879)

Reply:
My intuition is that if you need a root element for a stand alone component, as is the case with something like hint or error that could be used outside `Field`, `Box` is the go to.

`Text`, for me, is mainly a child element that can't stand alone. For me that would push label and legend into Box territory as well.

Reply:
@jeffbelser-okta @andrewberg-okta okay, that matches my thinking.

Would the exception be using `<Text>` to insert a pre-styled element? In that case it seems built to stand alone.

Reply:
@edburyenegren-okta what do you mean by using `<Text>` to insert a pre-styled element? I'm not sure I follow

Reply:
`<Text as="abbr" title="HyperText Markup Language>HTML</Text>`